bucket4j-spring-boot-starter-context/src/main/java/com/giffing/bucket4j/spring/boot/starter/context/constraintvalidations/UrlValidator.java

```diff
@@ -0,0 +1,29 @@
+package com.giffing.bucket4j.spring.boot.starter.context.constraintvalidations;
+
+import com.giffing.bucket4j.spring.boot.starter.context.UrlMapper;
+import jakarta.validation.ConstraintValidator;
+import jakarta.validation.ConstraintValidatorContext;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class UrlValidator
```

Comment:
Optional - what do you think?: Maybe something like UrlPatternMatcherValidator to give a hint, that not an URL is validated. Afterwards I should have name the property maybe url.pattern=.*